pyproject.toml

```diff
@@ -88,11 +91,8 @@ all = [
     "mpmath",
     "asdf-astropy>=0.3",
     "bottleneck",
-    "ipython>=4.2",
```

Comment:
Actually, given our discussion over in #16351, perhaps we shouldn't have "hidden" dependencies - code has a dependence on whether `ipython` is installed in `utils.console`.

But possibly that is best done separately - here, what we list is the dependencies needed for all astropy functionality.

Anyway, am happy either with this as-is, or with `ipython` put back.

Reply:
#16354 👀

Reply:
Any dependency we have should be listed exactly once, but it would indeed be better to keep `ipython` here and to delete the duplicate from the `test_all` set instead.

Reply:
Done.